marimbabot_planning/launch/marimbabot.launch

```diff
@@ -0,0 +1,4 @@
+<launch>
+    <node name="lilypond_to_hitpoint" pkg="marimbabot_planning" type="lilypond_to_hitpoint.cpp"/>
+    <node name="move_to_pose" pkg="marimbabot_planning" type="move_to_pose.cpp"/>
```

Comment:
Did you confirm that this works, because I don't think that the `.cpp` is right from compiled cpp binaries in the ROS context.

Reply:
sure, you are right

Reply:
I don't think this issue has been addressed.

Reply:
now it should be @Flova